src/Html2Text.php

```diff
@@ -263,10 +274,16 @@ static function iterateOverNode($node, $prevName = null, $in_pre = false, $is_of
 				break;
 
 			case "tr":
-			case "div":
 				// add one line
 				$output = "\n";
 				break;
+			case "div":
```

Comment:
Please put in an empty line between the break and the next case statement, so it's consistent with the rest of the file